app/facades/quotable_facade.rb

```diff
@@ -0,0 +1,14 @@
+class QuotableFacade
+  def self.top_results(query)
```

Comment:
Nice job taking into consideration that we want both the `:totalCount` and `quote attributes`. Here is a way that we can do that is a single method with a single call:
```
def self.quotes(query)
    json = QuotableService.top_results(query)
    quotes = json[:results].map do |quote_data|
      Quote.new(quote_data)
    end
    [json[:totalCount], quotes]
end
```
This will give us access to both things in a single call. And we can pass both of those to our controller as well.